docs/openapi_v1.0.0-alpha.yaml

```
@@ -1084,6 +1123,16 @@ components:
       description: |
         This UUID is generated for the initial `createReservation` request and MUST never change. This value MUST be tracked by both the Reseller and Booking Platform for locating this record.
       example: '7df49d62-57ad-44be-8373-e4c2fe7e63fe'
+    CalendarItem:
+      title: CalendarItem
+      type: object
+      properties:
```

Are there any required fields for this object? I'd suggest that both fields should be required if this object is present.

```suggestion
      required:
        - localDate
        - capacity
      properties:
```

---

Applied your suggestion